bgstally/debug.py

```diff
@@ -1,19 +1,22 @@
 import logging
 from os import path
 
-from config import appname
+from config import appname # type: ignore
 
 
 class Debug:
     logger: logging.Logger
 
-    def __init__(self, bgstally):
+    def __init__(self, bgstally, dev_mode: bool = False) -> None:
         # A Logger is used per 'found' plugin to make it easy to include the plugin's
         # folder name in the logging output format.
         # NB: plugin_name here *must* be the plugin's folder name as per the preceding
         #     code, else the logger won't be properly set up.
         Debug.logger = logging.getLogger(f'{appname}.{path.basename(bgstally.plugin_dir)}')
 
+        if dev_mode == False:
+            Debug.logger.setLevel(logging.INFO)
```

Comment:
I think this should be moved inside the `if` statement below, otherwise it will be overwritten by line 23. My understanding is that under normal circumstances that code will always execute as the logger has not already been set up.

```python
if dev_mode == False:
    level = logging.INFO  # So logger.info(...) is equivalent to print()
else:
    level = logging.DEBUG
```